src/test/java/core/basesyntax/service/impl/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
+    public static final List<FruitTransaction> VALID_TRANSACTION_LIST
+            = List.of(new FruitTransaction(
+            FruitTransaction.Operation.BALANCE, "banana", 20));
+    private static FruitTransactionParser transactionParser;
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void setUp() {
+        transactionParser = new FruitTransactionParserImpl();
+        fileReaderService = new FileReaderServiceImpl();
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#ensure-that-you-test-your-services-and-their-independently

Reply:
fixed